app/webhooks/solidus_bolt/handlers/credit_handler.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+module SolidusBolt
+  module Handlers
+    class CreditHandler < BaseHandler
+      def call
+        response_code = params[:data][:reference]
```

Comment:
Reading the payload I think it should be

`response_code = params[:data][:source_transaction][:reference]`